kitsune/tags/south_migrations/0001_add_buddyup_tags.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        categories = [
+            'email', 'clock', 'camera', 'audio', 'performance', 'radios', 'sms', 'calls',
+            'browser', 'storage', 'geolocation', 'keyboard', 'system-updates', 'language',
+            'marketplace', 'other'
+        ]
+
+        for category in categories:
+            name = 'category:{}'.format(category)
+            orm['taggit.tag'].objects.create(name=name, slug=name)
+
+    def backwards(self, orm):
+        orm['taggit.tag'].objects.filter(name__startswith='category:').delete()
```

Comment:
If we decide to keep the slug with the colon, we should change this to `slug__startswith` since we'd  never have a slug starting with 'category:' coming from the API (or anywhere else the user can create tags).